v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -146,5 +161,56 @@ def requires(self) -> ReferenceDataset:
             RAW_DATASET_PATH: 'https://mitimpact.css-mendel.it/cdn/MitImpact_db_3.1.3.txt.zip',
         },
     },
+    ReferenceDataset.hgmd: {
+        ENUMS: {'class': HGMD_CLASSES},
+        ReferenceGenome.GRCh37: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://seqr-reference-data-private/GRCh37/HGMD/HGMD_Pro_2023.1_hg19.vcf.gz',
+        },
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://seqr-reference-data-private/GRCh38/HGMD/HGMD_Pro_2023.1_hg38.vcf.gz',
+        },
+    },
+    ReferenceDataset.gnomad_exomes: {
+        MODULE_NAME: 'gnomad',
+        ReferenceGenome.GRCh37: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/2.1.1/ht/exomes/gnomad.exomes.r2.1.1.sites.ht',
+        },
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/4.1/ht/exomes/gnomad.exomes.v4.1.sites.ht',
+        },
+    },
+    ReferenceDataset.gnomad_genomes: {
+        MODULE_NAME: 'gnomad',
+        ReferenceGenome.GRCh37: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/2.1.1/ht/genomes/gnomad.genomes.r2.1.1.sites.ht',
+        },
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/4.1/ht/genomes/gnomad.genomes.v4.1.sites.ht',
+        },
+    },
+    ReferenceDataset.gnomad_qc: {
```

Comment:
Shouldn't this be a `ReferenceDatasetQuery` not a `ReferenceDataset`?

Reply:
I don't think so because its dependency is itself: https://github.com/broadinstitute/seqr-loading-pipelines/blob/6206bdcf20f5ea4519872e964a6713c1dd308466/v03_pipeline/lib/model/cached_reference_dataset_query.py#L31

Reply:
Yes that's correct that it's still special and would be included with the code as is.  I also think we can fix when we write the code that utilizes `for_reference_genome_dataset_type`.